assets/src/scss/layout/_navbar.scss

```diff
@@ -75,6 +74,15 @@ $navbar-default-height: 60px;
       top: 32px;
     }
   }
+
+  a, .btn, input {
+    &:focus {
+      --top-navigation--ui-controls-- {
```

Comment:
I made this into a single declaration, so that they can all be changed in one go if the top navigation color changes.